README.md

```diff
@@ -1,24 +1,47 @@
-# README
+# VocabiApp
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## What is this app?
 
-Things you may want to cover:
+This a **vocabulary trainer for foreign languages**, the game is as follows,
+you sign up (just using you email), specify your **original** language and your
+**target** language (the one on that you want to improve your vocabulary)
+pick a *part of speech* category, i.e, pronouns, verbs, etc. After selecting
+the category, click on a level, inside the level you'll need to translate the
+displayed words into your original language, answering correctly gives you
+points! The user with more points will appear at the top of the ranking list of
+the home page. Pretty straightforward, isn't it? Well, that kind of explains
+everything about the app, have a good time while using it! And, if you want to,
+collaborate to its development!
 
-* Ruby version
+## App details
 
-* System dependencies
+### System dependencies
 
-* Configuration
+* `ruby` (2.6.5)
+* `bundler`
+* `sqlite3` (development)
+* `postgresql` (production)
 
-* Database creation
+### Development environment setup
 
-* Database initialization
+After cloning the repository and changing your current directory
+to the one representing the app, do the following.
 
-* How to run the test suite
+```console
+bundle install && rails db:migrate && rails db:seed
+```
 
```

Comment:
On my env I also need to execute `yarn install --check-files`, do u also needed it?

Reply:
Yes, I forgot it, now should be pointed out.